internal/balancergroup/balancergroup.go

```diff
@@ -411,20 +409,6 @@ func (bg *BalancerGroup) cleanupSubConns(config *subBalancerWrapper) {
 	}
 }
 
-// connect attempts to connect to all subConns belonging to sb.
-func (bg *BalancerGroup) connect(sb *subBalancerWrapper) {
-	bg.incomingMu.Lock()
-	defer bg.incomingMu.Unlock()
-	if bg.incomingClosed {
-		return
-	}
-	for sc, b := range bg.scToSubBalancer {
-		if b == sb {
-			sc.Connect()
-		}
-	}
-}
-
```

Comment:
i removed here, it fails CI-test because it's unused.
![image](https://github.com/user-attachments/assets/65ef782b-970a-45ae-98e4-719fdbb4a5a2)